sdk/resourcemanager/Azure.ResourceManager/src/Common/Custom/Models/EncryptionProperties.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.Models
+{
+    // this class here is to keep the class public and add the EditorBrowsableNever attribute
+    // this class is exposed in resourcemanager by accident, now we hide it in resourcemanager
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Add the Obsolete as warning attribute here as well.
`[Obsolete("This type is obsolete and will be removed in a future release.", false)]`